app/src/generator/Radii/Radii.tsx

```diff
@@ -16,19 +16,57 @@ const Radii = () => {
   const [currValues, setCurrValues] = useState<Map<string, string | number>>(
     new Map<string, string | number>()
   );
+  const currValuesRef = useRef<Map<string, string | number>>(currValues);
```

Comment:
We could type the radius better, with `Radius`* instead of `string` (here and everywhere else)

```tsx
type Radius = keyof HvThemeTokens["radii"];
```
^ an helper alias

```suggestion
  const currValuesRef = useRef<Map<Radius, string | number>>(currValues);
```

Actually, here the type can be inferred
```suggestion
  const currValuesRef = useRef(currValues);
```